src/framework/components/model/component.js

```diff
@@ -47,6 +47,8 @@ import { Component } from '../component.js';
  * @property {boolean} lightmapped If true, this model will be lightmapped after using lightmapper.bake().
  * @property {number} lightmapSizeMultiplier Lightmap resolution multiplier.
  * @property {boolean} isStatic Mark model as non-movable (optimization).
+ * @property {pc.BoundingBox} aabb If set, the bounding box is used as a bounding box for visibility culling of attached mesh instances. This is an optimization,
```

Comment:
might be clearer if this is also called overrideAabb?

Reply:
I had it originally called that, but decided just aabb is more suitable

Reply:
Yeah, I'd probably agree. My order of preference would be:

1. `aabb`
2. `customAabb`
3. `overrideAabb`